app/src/main/res/drawable/seek_bar_thumb.xml

```diff
@@ -0,0 +1,8 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+    <size
+        android:height="14dp"
+        android:width="14dp"
+        />
+    <solid android:color="@color/white"/>
+</shape>
```

Comment:
favorite - end of file

Reply:
I've relayed on ktlint this time, but no :(